docs/managing_deis/backing_up_data.rst

```diff
@@ -217,6 +217,13 @@ use in the ``export`` command should correspond to the IP of the host machine wh
 
 That's it! The cluster should be fully restored.
 
+Tools
+-----
+
+Various community members have developed tools to assist in automating the backup and restore process outlined above,
```

Comment:
I would split this into a separate sentence, but otherwise this is exactly what I was thinking. :+1: